components/hab/src/cli.rs

```diff
@@ -291,10 +291,15 @@ pub fn get() -> App<'static, 'static> {
                     Environment variables (those starting with 'pkg_') that are set will be used \
                     in the generated plan")
                 (aliases: &["i", "in", "ini"])
-                (@arg PKG_NAME: +takes_value "Name for the new app.")
+                (@arg PKG_NAME: +takes_value "Name for the new app")
                 (@arg ORIGIN: --origin -o +takes_value "Origin for the new app")
-                (@arg NO_CALLBACKS: --nocallbacks -f
-                    "Do not include callback functions in template")
+                (@arg WITH_DOCS: --withdocs "Include plan options documentation")
```

Comment:
Think you could make this `--with-docs` rather than `--withdocs`? That will match some of the other CLI arguments in Habitat like in the [Supervisor](https://github.com/habitat-sh/habitat/blob/182ef7b4035aea078ad98018257e5190f759f18b/components/sup/src/main.rs#L197). You should be able to do this with `--("with-docs")` using the clap macro DSL.